ide/lsp.client/src/org/netbeans/modules/lsp/client/LSPBindings.java

```diff
@@ -463,12 +496,10 @@ public InitializeResult getInitResult() {
 
     @SuppressWarnings("AccessingNonPublicFieldOfAnotherObject")
     public static synchronized void addBackgroundTask(FileObject file, BackgroundTask task) {
-        RequestProcessor.Task req = WORKER.create(() -> {
-            LSPBindings bindings = getBindings(file);
-
-            if (bindings == null)
-                return ;
-
+        LSPBindings bindings = getBindings(file);
+        if (bindings == null)
+            return ;
```

Comment:
Please use braces (yeah nitpick):

```suggestion
        if (bindings == null) {
            return ;
        }
```